package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "eslint": "npx eslint *.js && npx eslint *.jsx && npx eslint *.tsx"
```

Comment:
I think we can set eslint to pick up all the extensions in one go

`npx eslint --ext .js,.jsx,.tsx`